modules/lfs/server.go

```diff
@@ -233,6 +233,12 @@ func PostHandler(ctx *context.Context) {
 		return
 	}
 
+	if setting.LFS.MaxFileSize > 0 && rv.Size > setting.LFS.MaxFileSize {
+		log.Info("Denying LFS upload of size %d (>%d) to %s/%s", rv.Size, setting.LFS.MaxFileSize, rv.User, rv.Repo)
```

Comment:
```suggestion
		log.Info("Denied LFS upload of size %d to %s/%s because of LFS_MAX_FILE_SIZE=%d", rv.Size, rv.User, rv.Repo, setting.LFS.MaxFileSize)
```